styles/base/index.js

```diff
@@ -0,0 +1,20 @@
+var fs = require('fs')
+var path = require('path')
+var compile = require('micro-css')
+var result = ''
+var additional = ''
+
+fs.readdirSync(__dirname).forEach(function (file) {
+  if (/\.mcss$/i.test(file)) {
+    result += fs.readFileSync(path.resolve(__dirname, file), 'utf8') + '\n'
+  }
+
+  if (/\.css$/i.test(file)) {
+    additional += fs.readFileSync(path.resolve(__dirname, file), 'utf8') + '\n'
+  }
+})
+
+const flatpickr = require.resolve('flatpickr/dist/flatpickr.css')
```

Comment:
I noticed that flatpickr styles are included in the `base` css as well as `dark` and `light`. Should just be in the `base`.

Reply:
Done, removed from dark and light. :+1: